person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  def initialize(name, age = 'unknown', parent_permission = 'true')
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  # getter
+  attr_accessor :name, :age
+  attr_reader :parent_permission
```

Comment:
- [ ] As mentioned by the previous reviewer, kindly make sure  the `id` is only able to be retrieved so only the `Getter` is defined to it and not the `parent_permittion` (i.e. `attr_reader :id`)